src/Datagrid/Pager.php

```diff
@@ -39,25 +36,11 @@ public function computeNbResult()
         return $countQuery->count()->getQuery()->execute();
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function getResults()
     {
         return $this->getQuery()->execute();
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function getQuery()
```

Comment:
isnt this bc break?

Reply:
[It's already defined in the base class as it is](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Datagrid/Pager.php#L564)